### PR TITLE
Handled isFailoverEnabled in selectFinalResult controller

### PR DESCRIPTION
**Pull request by balavor**
## Changes
If failover feature is disabled, we will handle only one final result.

@VerizonAdPlatforms/video-partner-sdk-developers: Please review.

### CODE REVIEW

File: sources/advertisements/VRM New Core/Controllers/VRMSelectFinalResultController.swift [hunk 1]
```diff
@@ -13,25 +13,37 @@ final class VRMSelectFinalResultController {
         self.dispatch = dispatch
     }
     
-    func process(with state: PlayerCore.State) {
+    func process(with state: PlayerCore.State, isFailoverEnabled: Bool) {
```
Comment (VladyslavAnokhin) [inline sources/advertisements/VRM New Core/Controllers/VRMSelectFinalResultController.swift]: You can pass `isFailoverEnabled` value in `init`

File: sources/advertisements/VRM New Core/Controllers/VRMSelectFinalResultController.swift [hunk 2]
```diff
@@ -13,25 +13,37 @@ final class VRMSelectFinalResultController {
...
+            return
+        }
         guard isMaxAdSearchTimeoutReached == false,
-            finalResult == nil,
-            let currentGroup = currentGroup else {
+            finalResult.successResult == nil else {
                 return
         }
         
+        if firedResults.isEmpty == false &&
```
Comment (VladyslavAnokhin) [inline sources/advertisements/VRM New Core/Controllers/VRMSelectFinalResultController.swift]: guard

File: sources/advertisements/VRM New Core/Controllers/VRMSelectFinalResultController.swift (hunk 1 above)
Comment (balavor) [reply sources/advertisements/VRM New Core/Controllers/VRMSelectFinalResultController.swift]: Done

File: sources/advertisements/VRM New Core/Controllers/VRMSelectFinalResultController.swift (hunk 2 above)
Comment (balavor) [reply sources/advertisements/VRM New Core/Controllers/VRMSelectFinalResultController.swift]: Nope. Discussed offline

**Review by VladyslavAnokhin — APPROVED** [review]

**Review by VladyslavAnokhin — APPROVED** [review]

**Pull request merged**